draft-wood-tls-external-psk-importer.md

```diff
@@ -97,20 +100,37 @@ supported HashAlgorithm 'hash', the importer constructs an ImportedIdentity stru
    } ImportedIdentity;
 ~~~
 
+[[TODO: An alternative deisgn might combine label and hash into the same field so that future
```

Comment:
Nit: s/deisgn/design/

Yet another option would be that your protocol picks a single "label", and then anything following it is your business. For TLS 1.3, we use HashAlgorithm. Future TLS versions might have other opinions.